### ISSUE TITLE
some parse problems

### ISSUE THREAD

**Issue by hacknlove** [fuhrmanator/GIFT-grammar-PEG.js#27]
`->`

```
// question: 882007  name: iebT26nuevo_P012
::iebT26nuevo_P012::¿Cómo se cambia la orientación de la página en Word 2003 y 2007?\:{
	~todas las versiones de Word cambian la orientación de página igual (archivo->diseño de página)
	~-archivo->configurar->orientación ó -archivo->configurar->diseño de página
	=-archivo-configurar página ó -diseño de página-orientación
}
```

**Comment by fuhrmanator** [comment]
Thanks for posting these issues. You're welcome to see if you can solve the problem in the grammar with PEG, otherwise I will eventually look at them when my schedule permits. Cheers!

**Comment by fuhrmanator** [comment]
Hint: This bug is due to the `->` in the answers.

The current grammar treats `->` as a control character to distinguish `RichText` (for matching questions). 

I think a true matching question as *all* choices that start with `=` and have a `->` inside them. But I'm not sure how to do that with PEG, at least in the current grammar design.

**Comment by fuhrmanator** [comment]
Fixed with #32
